cmd/test.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/exercism/cli/workspace"
+	"github.com/spf13/cobra"
+)
+
+var testCmd = &cobra.Command{
+	Use:     "test",
+	Aliases: []string{"t"},
+	Short:   "Run the exercise's tests.",
+	Long: `Run the exercise's tests.
+
+	Run this command in an exercise's root directory.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return runTest(args)
+	},
+}
+
+func runTest(args []string) error {
+	track, err := getTrack()
+	if err != nil {
+		return err
+	}
+
+	testConf, ok := workspace.TestConfigurations[track]
+
+	if !ok {
+		return fmt.Errorf("test handler for the `%s` track not yet implemented. Please see HELP.md for testing instructions", track)
+	}
+
+	command, err := testConf.GetTestCommand()
+	if err != nil {
+		return err
+	}
+	cmdParts := strings.Split(command, " ")
+
+	// pass args/flags to this command down to the test handler
+	if len(args) > 0 {
+		cmdParts = append(cmdParts, args...)
+	}
+
+	fmt.Printf("--> %s\n", strings.Join(cmdParts, " "))
```

Comment:
I really like the logging! My only concern is that it might be _too_ subtle for the student to notice. Maybe we could make it more explicit? To instead of `--> <command>`, something like `Running tests via: <command>` or `Running test command: <command>`

Reply:
Sure! updated to:

```
Running tests via `<command>`\n\n
```

which makes it clear exactly what's running and gives it some space to breathe